sdk/communication/AzureCommunicationChat/README.md

```diff
@@ -238,7 +237,7 @@ Use the `listChatThreads` method to retrieve a list of threads.
 ```swift
 import AzureCore
 let options = ListChatThreadsOptions(maxPageSize: 1)
-client.listChatThreads(withOptions: options) { result, _ in
+chatClient.listChatThreads(withOptions: options) { result, _ in
```

Comment:
Since this is a "ChatClient", would it be simpler to call this method `listThreads`?

Reply:
Also just verifying that there's an overload where the customer can omit the `withOptions` parameter

Reply:
There wouldn't be an overload per se, but the signature should be `withOptions options: BlahOptions? = nil, ...`

Reply:
^this is correct, and so the customer can call listThreads() without options.